torch/csrc/distributed/rpc/request_callback_impl.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <torch/csrc/distributed/rpc/message.h>
+#include <torch/csrc/distributed/rpc/request_callback.h>
+#include <torch/csrc/distributed/rpc/rpc_command_base.h>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+class TORCH_API RequestCallbackImpl : public RequestCallback {
+ public:
+  Message processMessage(const Message& request) override;
+
+ private:
+  std::unique_ptr<RpcCommandBase> processRpc(
+      RpcCommandBase* rpc,
```

Comment:
Raw pointer here as well. I am bit timid of mixing smart pointers and raw pointers. What's the reason for using raw pointer here?